src/components/Reply/Content/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import styled, { css } from 'styled-components';
+
+import { gray500, gray800 } from '../../../core/Colors';
+import { body2 } from '../../../core/TextStyles';
+import { TextButton } from '../../Button';
+
+interface Props {
+  useLineClamp: boolean;
+  maxLine: number;
+  readMoreText: string;
+  hideText: string;
+  onClick?: React.MouseEventHandler<HTMLDivElement>;
+}
+
+export const ReplyContent: FC<Props> = React.memo(props => {
+  const { children, useLineClamp, maxLine, readMoreText, hideText, onClick } = props;
+
+  const [lineClamped, setLineClamped] = useState(true);
+  const [lineClampable, setLineClampable] = useState(false);
+
+  const contentRef = React.useRef<HTMLDivElement | null>(null);
+
+  const getContentClamped = useCallback(() => {
+    const { current } = contentRef;
+
+    if (children && current) {
+      return current.scrollHeight > current.clientHeight;
+    }
+    return false;
+  }, [children]);
+
+  const calculateClamped = useCallback(() => {
+    const initialLineClampable = useLineClamp && getContentClamped();
+    setLineClampable(initialLineClampable);
+    setLineClamped(initialLineClampable);
+  }, [useLineClamp, getContentClamped]);
+
+  useEffect(() => {
+    calculateClamped();
+  }, [calculateClamped]);
+
+  const handleToggleClampButton = useCallback(
+    (event: React.MouseEvent<Element, MouseEvent>) => {
+      setLineClamped(!lineClamped);
+      event.preventDefault();
+      event.stopPropagation();
+    },
+    [lineClamped]
+  );
+
+  return (
+    <>
+      <Content useLineClamp={lineClamped} maxLine={maxLine} ref={contentRef} onClick={onClick}>
+        {children}
+      </Content>
+      {lineClampable && (
+        <ClampToggleButton onClick={handleToggleClampButton}>{lineClamped ? readMoreText : hideText}</ClampToggleButton>
+      )}
+    </>
+  );
+});
+
+const Content = styled.div<{ useLineClamp: boolean; maxLine: number }>`
+  ${body2};
+  display: -webkit-box;
+  -webkit-box-align: baseline;
```

Comment:
```suggestion
  display: box;
  box-align: baseline;
```
요렇게 쓰면 autoprefixer같은게 prefix같은거 안달아주려낭?? cc. @chiabi 
ui에 설정이 안되있으려남

Reply:
요거 styled-components의 css pre-processor인 stylus에서 auto-prefix 붙여주는걸로 알아
https://github.com/stylus/stylus

Reply:
styled-component가 autoprefixer를 지원하긴 하는데
> **Automatic vendor prefixing:** write your CSS to the current standard and let styled-components handle the rest.
You get all of these benefits while still writing the CSS you know and love, just bound to individual components.

autoprefixer에서 접두사를 붙이지 않는 예외의 경우가 몇가지 있음다.
👉 https://github.com/postcss/autoprefixer/issues/1014

<img width="393" alt="스크린샷 2020-06-15 오후 4 16 54" src="https://user-images.githubusercontent.com/15900134/84631764-e42e1780-af28-11ea-9947-19b795f4fc39.png">

Reply:
호웃..😥